### PR TITLE
Fixit ticket 530

### PR DESCRIPTION
**Pull request by kildre**
Made updates to the UI 
## Reference Issues or PRs

Added loading state to creat app button, updates to the chips size, tweaks ot the text fields
## What does this implement/fix?

_Put a `x` in the boxes that apply_

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x ] New feature (non-breaking change which adds a feature)
- [ ] Breaking change (fix or feature that would cause existing features not to work as expected)
- [ ] Documentation Update
- [x] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build related changes
- [ ] Other (please describe):

## Testing

- [x] Did you test the pull request locally?
- [ ] Did you add new tests?

#### Text styling

- [x] The content is written with [plain language](https://www.plainlanguage.gov/guidelines/) (where relevant).
- [ ] If there are headers, they use the proper header tags (with only one level-one header: `H1` or `#` in markdown).
- [ ] All links describe where they link to (for example, check the [Nebari website](https://nebari.dev/)).
- [ ] This content adheres to the Nebari style guides.

#### Non-text content

- [ ] All content is represented as text (for example, images need alt text, and videos need captions or descriptive transcripts).
- [ ] If there are emojis, there are not more than three in a row.
- [ ] Don't use [flashing GIFs or videos](https://www.w3.org/TR/UNDERSTANDING-WCAG20/seizure-does-not-violate.html).
- [ ] If the content were to be read as plain text, it still makes sense, and no information is missing.

## Any other comments?
image

removed-horizontal-separators

medium_chips

### CODE REVIEW

File: ui/package.json [hunk 1]
```diff
@@ -19,6 +19,7 @@
     "@emotion/react": "^11.11.4",
     "@emotion/styled": "^11.11.5",
     "@mui/icons-material": "^5.15.15",
+    "@mui/lab": "^5.0.0-alpha.170",
```
Comment (jbouder) [inline ui/package.json:22]: Is there a non-alpha version? Maybe something more stable?

File: ui/src/components/app-card/app-card.css [hunk 2]
```diff
@@ -21,7 +25,7 @@
   height: 130px;
   padding: 33px 0;
   overflow: hidden;
-  background: linear-gradient(90deg, #fff 0%, #ece9e6 95.83%);
+  /* background: linear-gradient(90deg, #fff 0%, #ece9e6 95.83%); */
```
Comment (jbouder) [inline ui/src/components/app-card/app-card.css:28]: Can we remove this?

File: ui/src/components/app-card/app-card.css [hunk 3]
```diff
@@ -158,6 +162,22 @@
...
 
+.card-header {
+  display: flex;
+  align-items: center;
+}
+
+.inline.relative.iconic {
+  display: flex;
+  align-items: center;
+  margin-right: 8px; /* Adjust margin as needed */
```
Comment (jbouder) [inline ui/src/components/app-card/app-card.css:173]: Remove this comment

File: ui/src/components/app-card/app-card.tsx [hunk 4]
```diff
@@ -170,7 +170,11 @@ export const AppCard = ({
   };
 
   return (
-    <Box className="card" id={`card-${id}`} tabIndex={0}>
+    <Box
+      className={`card ${isAppCard ? '' : 'service'}`}
```
Comment (jbouder) [inline ui/src/components/app-card/app-card.tsx:174]: What is this doing?

File: ui/src/pages/server-types/server-types.css [hunk 5]
```diff
@@ -14,3 +14,22 @@
...
+    x: -100%;
+  }
+  100% {
+    x: 100%;
+  }
+}
+
+.mask-rect {
+  animation: move-mask 2s linear infinite;
+}
```
Comment (jbouder) [inline ui/src/pages/server-types/server-types.css:35]: Is this needed when using the Loading Button?

**Review by jbouder — CHANGES_REQUESTED** [review]
A few changes. Plus seeing some formatting issues when loading the app
Screenshot 2024-05-22 at 4 13 01 PM

File: ui/src/components/app-card/app-card.tsx (hunk 4 above)
Comment (kildre) [reply ui/src/components/app-card/app-card.tsx:174]: Appliging a 'service' class to the service cards for unique styling ont he service ards. The service cards get a different height than the app cards.

File: ui/package.json (hunk 1 above)
Comment (kildre) [reply ui/package.json:22]: Nope looks like it's all alhpa, and a canary.
https://www.npmjs.com/package/@mui/lab?activeTab=versions

File: ui/src/components/app-card/app-card.css (hunk 3 above)
Comment (kildre) [reply ui/src/components/app-card/app-card.css:173]: Removing styles all together.

File: ui/src/pages/server-types/server-types.css (hunk 5 above)
Comment (kildre) [reply ui/src/pages/server-types/server-types.css:35]: Styles not needed. Removed

File: ui/src/components/app-card/app-card.css (hunk 2 above)
Comment (kildre) [reply ui/src/components/app-card/app-card.css:28]: I put it back in, I was testing if it was needed, it's a fall back background gradient so leaving as was.

**Comment by kildre** [comment]
> A few changes. Plus seeing some formatting issues when loading the app Screenshot 2024-05-22 at 4 13 01 PM

Made updates, alignment back to way it should be.

**Review by jbouder — APPROVED** [review]
Looks great!

**Pull request merged**